azkaban-solo-server/src/main/resources/quartz_schema.sql

```diff
@@ -0,0 +1,249 @@
+-- Thanks to Amir Kibbar and Peter Rietzler for contributing the schema for H2 database, 
+-- and verifying that it works with Quartz's StdJDBCDelegate
+--
+-- Note, Quartz depends on row-level locking which means you must use the MVCC=TRUE 
+-- setting on your H2 database, or you will experience dead-locks
+--
+--
+-- In your Quartz properties file, you'll need to set 
+-- org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.StdJDBCDelegate
+
+CREATE TABLE QRTZ_CALENDARS (
+  SCHED_NAME VARCHAR(120) NOT NULL,
+  CALENDAR_NAME VARCHAR (200)  NOT NULL ,
+  CALENDAR IMAGE NOT NULL
+);
+
+CREATE TABLE QRTZ_CRON_TRIGGERS (
```

Comment:
Can you please use CREATE TABLE IF NOT EXISTS? Even though it is for solo server and we expect that these table won't be existing. Can you please add it for all the tables?

Reply:
This is a verbatim copy of the schema definition found in the quartz source code. I'll look into such a change, though the add key statements should also be idempotent. As is, the errors are logged at a low level and the ignored. This case is actually common - any startup of the solo-server past the first one.